internal/testcase/testcase_test.go

```diff
@@ -67,28 +67,62 @@ func TestNew(t *testing.T) {
 				IgnoredFields: []string{"@version"},
 			},
 		},
-		// No handle input with bracket notation when codec is line
+		// Ignore bracket notation in TCS input lines when codec is line
 		{
 			input: `{"input": ["{\"[test][path]\": \"test\"}"]}`,
 			expected: TestCaseSet{
 				Codec:         "line",
-				InputLines:    []string{"{\"[test][path]\": \"test\"}"},
+				InputLines:    []string{`{"[test][path]": "test"}`},
 				IgnoredFields: []string{"@version"},
 				InputFields:   logstash.FieldSet{},
 				Events:        []logstash.FieldSet{{}},
 			},
 		},
-		// handle input with bracket notation when codec is json_lines
+		// Transform bracket notation in TCS input lines when codec is json_lines
 		{
 			input: `{"input": ["{\"[test][path]\": \"test\"}"], "codec": "json_lines"}`,
 			expected: TestCaseSet{
 				Codec:         "json_lines",
-				InputLines:    []string{"{\"test\":{\"path\":\"test\"}}"},
+				InputLines:    []string{`{"test":{"path":"test"}}`},
 				IgnoredFields: []string{"@version"},
 				InputFields:   logstash.FieldSet{},
 				Events:        []logstash.FieldSet{{}},
 			},
 		},
+		// Ignore bracket notation in TC input lines when codec is line
```

Comment:
```suggestion
		// Ignore bracket notation in TCS input lines when codec is line.
```

Reply:
No, I really meant TC. This testcase uses the bracket notation in the testcases slices rather than at the top level, the TCS. We could look into making the phrasing clearer to distinguish these two cases, but the two TCS cases will be deleted when the top-level `input` key is removed in a follow-up commit.